app/models/alert.rb

```diff
@@ -47,6 +47,8 @@ class Alert < ApplicationRecord
   enum enough_data: [:enough, :not_enough, :minimum_might_not_be_accurate], _prefix: :data
   enum relevance: [:relevant, :not_relevant, :never_relevant], _prefix: :relevance
 
+  scope :without_exception, -> { joins(:alert_type).joins('LEFT OUTER JOIN school_alert_type_exceptions ON school_alert_type_exceptions.school_id = alerts.school_id AND school_alert_type_exceptions.alert_type_id = alert_types.id').where(school_alert_type_exceptions: { school_id: nil }) }
```

Comment:
Prefer `without_exceptions`, "without exception" sounds like the opposite of what we're trying to do!